app/jobs/create_civicpy_cache_pkl.rb

```diff
@@ -1,7 +1,13 @@
 class CreateCivicpyCachePkl < ActiveJob::Base
-  def perform
+  attr_reader :recurring
+
+  after_perform do |job|
+    job.reschedule if job.recurring
+  end
+
+  def perform(recurring = true)
+    @recurring if recurring
```

Comment:
Same here - looks good otherwise though. Good to merge after fixing!

Reply:
```suggestion
    @recurring = recurring
```